### PR TITLE
doc updates for ipv6

### PR DESCRIPTION
**Pull request by aaha97**

### CODE REVIEW

**Review by akshayhavile — APPROVED** [review]

File: docs/ipv6-support.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## Overview
```
Comment (swathinsankaran) [inline docs/ipv6-support.md:1]: Is this a new document that we are adding in 1.9.2?

File: docs/ipv6-support.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+| Use case   |      Dual stack support      |  comments |
+|----------|---------------|-------|
+| Frontend VIP |  SUPPORTED | Customers can choose to have either v6only or dual VIPs (v4, v6) for the Virtual services |
+| Backend (Pod IPs) |    V6ONLY   |   AKO will add either v4 or v6 addresses to the pools based on AKO configuration ('ipFamily'). AKO does not support a mix of v4 and v6 addresses |
+| K8s Nodes | V6ONLY |    AKO configures routes to the pod IPs via Node IP. AKO will choose v4 or a v6 IP based on ipFamily attribute in AKO config. For NodePort mode, AKO will choose either v4 or v6 IPs of Nodes based on the ipFamily attribute in AKO config |
+| Avi Controller IP | V4ONLY | AKO only supports v4 IPs to talk to Avi controller mgmt interface. AKO pod needs IPv4 connectivity to Avi controller mgmt interface |
+| K8s API server | V4ONLY | AKO only supports IPv4 to talk to K8s API server. v6 is not supported |
+
+### Support Matrix
+Cloud: vCenter
```
Comment (swathinsankaran) [inline docs/ipv6-support.md:15]: please add this in a table for better readability.

**Review by swathinsankaran — APPROVED** [review]
Overall, the document looks good to me.

File: docs/ipv6-support.md (hunk 2 above)
Comment (aaha97) [reply docs/ipv6-support.md]: updated

File: docs/ipv6-support.md (hunk 1 above)
Comment (aaha97) [reply docs/ipv6-support.md:1]: yes, this is the support matrix for IPv6

Comment (akshayhavile) [reply docs/ipv6-support.md:1]: @aaha97 : Please mention end to end support is there only for L7.

File: docs/values.md [hunk 3]
```diff
@@ -89,15 +89,15 @@ The `blockedNamespaceList` lists the Kubernetes/Openshift namespaces blocked by
...
-### AKOSetttings.istioEnabled (Tech Preview)
+### AKOSetttings.istioEnabled
 
 AKO can be deployed in Istio environment. Setting this to `true` indicates to AKO that the environment is Istio. Default value is `false`.
 
-### AKOSetttings.ipFamily (Tech Preview)
+### AKOSetttings.ipFamily
 
-`V6` is currently supported only for `vCenter` cloud with `calico` CNI.
+`V6` is currently supported only for `vCenter` cloud with `calico` and `antrea` CNI.
```
Comment (swathinsankaran) [inline docs/values.md:98]: It would be better to add a link to the `support matrix` here.

File: docs/ipv6-support.md (hunk 1 above)
Comment (aaha97) [reply docs/ipv6-support.md:1]: added statement on end to end support

File: docs/values.md (hunk 3 above)
Comment (aaha97) [reply docs/values.md]: added link references to the support matrix doc

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**